packages/patternfly-4/react-core/src/components/Alert/AlertActionCloseButton.js

```diff
@@ -20,11 +20,11 @@ const defaultProps = {
   onClose: () => undefined
 };
 
-const AlertActionCloseButton = ({ className, onClose, 'aria-label': ariaLabel, title, variantLabel, ...props }) => (
+const AlertActionCloseButton = ({ className, onClose, variantLabel, 'aria-label': ariaLabel, title, variant, ...props }) => (
```

Comment:
I see an issue with the aria-label that's coming through. I'm not sure if it's because of this line or somewhere else, but... 

When I compare the html for the Close button,
this is what I see in [the first alert example in the PR preview](https://1731-pr-patternfly-react-patternfly.surge.sh/patternfly-4/components/alert/):
```
<button aria-label="Close undefined alert: Info alert title" class="pf-c-button pf-m-plain" type="button">
```
and this is [what is there now](https://patternfly-react.surge.sh/patternfly-4/components/alert):
```
<button aria-label="Close Success alert: Success alert title" class="pf-c-button pf-m-plain" type="button">
```

Notice that the aria-label includes "undefined alert" whereas the current build has "Success alert".

Reply:
So is it good now? I (think) I just fixed it in the latest commit that's preview finished a minute ago!